src/snapshot/Cargo.toml

```diff
@@ -9,6 +9,7 @@ serde_derive = ">=1.0.27"
 bincode = "1.2.1"
 utils = { path="../utils" }
 snapshot_derive = { path = "../snapshot_derive" }
+crc64 = "1.0.0"
```

Comment:
Have you considered other checksumming options? The [Fletcher](https://en.wikipedia.org/wiki/Fletcher%27s_checksum) checksum should outperform CRC by half, is trivial to implement, but does sacrifice on error prediction. However, since this code is less error prone than a networking stack and the snapshot must have passed through several layers of error checking before Firecracker deserializes it, I think it's an acceptable risk. In line with that thought, I think we could get away with a simple xor.

[This paper](https://users.ece.cmu.edu/~koopman/pubs/maxino09_checksums.pdf) has some very useful data on checksums.